scrapy_crawlera/middleware.py

```diff
@@ -222,18 +221,17 @@ def _is_crawlera_header(self, header_name):
         header_name = header_name.decode('utf-8').lower()
         return header_name.startswith(self.header_prefix.lower())
 
-    def _set_crawlera_default_headers(self, request):
+    def _set_default_crawlera_headers(self, request):
         for header, value in self._headers:
             if value is None:
                 continue
             request.headers.setdefault(header, value)
         lower_case_headers = [
             header.decode('utf-8').lower() for header in request.headers
         ]
-        if all(h.lower() in lower_case_headers for h in self.conflicting_headers):
-            logging.warn(
-                'The headers %s are conflicting on request %s. X-Crawlera-UA '
-                'will be ignored. Please check https://doc.scrapinghub.com/cr'
-                'awlera.html for more information'
-                % (str(self.conflicting_headers), request.url)
+        if 'X-Crawlera-Profile'.lower() in lower_case_headers:
+            logging.debug(
```

Comment:
@kmike this message is not shown once, it is shown per request.

Reply:
I think it is ok to show the message per request (discussed [here](https://github.com/scrapy-plugins/scrapy-crawlera/pull/58#discussion_r218315208), but it had to be `debug`, `warning` was too verbose.

Reply:
I'd still do it differently :) The problem with logging.debug is that it is common to set LOG_LEVEL to INFO, as debug is too verbose in many-many cases, mostly because it logs scraped items. For example, on Scrapy Cloud LOG_LEVEL in INFO by default. In such cases, after this change we won't be reporting an issue with user code, so it can go unnoticed. As an user I'd expect such issues to be reported to me with LOG_LEVEL=INFO or even LOG_LEVEL=WARNING.

What do you think about having both warning.warn call and logging.debug? I.e. print the problem regardless of user logging option, but do it once, and then print URLs with issues, but only if log level is set to debug?